openstack-k8s-operators/osp-director-dev-tools
# Add OSP compute nodes to overcloud

There are some req at the moment:
until the ipam is integrated in the operator it is expected that
- the available workers to be computes are ostest-worker-3 and
ostest-worker-4
- the workers IPs are hard coded in deployed-server-port-map.yaml
because of that it is expected that they come up in the order
ostest-worker-3, then ostest-worker-4. If the order changes the
IP the baremetalset identifies from findIPs is different at deployment
might fail.

Follow up required for:
- use persistent storage for rendered deployment files + clouds.yaml
- use local OCP registry for overcloud images
- run the overcloud deploy via a deployment job
